pymc3/distributions/continuous.py

```diff
@@ -141,22 +286,85 @@ class Normal(Continuous):
     """
     def __init__(self, mu=0.0, tau=None, sd=None, *args, **kwargs):
         super(Normal, self).__init__(*args, **kwargs)
-        self.mean = self.median = self.mode = self.mu = mu
-        self.tau, self.sd = get_tau_sd(tau=tau, sd=sd)
+        self.mu, self.tau = self.parametrize(mu, tau, sd)
+        self.mean = self.median = self.mode = self.mu
         self.variance = 1. / self.tau
 
-    def logp(self, value):
+    def parametrize(self, mu=None, tau=None, sd=None):
+        try:
+            tau, sd = get_tau_sd(tau, sd)
+        except ValueError as err:
+            if check:
+                raise err
+        return mu, tau
+    
+    def pdf(self, x, point=None):
+        """
+        The probability density function (pdf) of the Normal distribution.
+        
+        Parameters
+        ----------
+            x : float
+                The value at which to evaluate the pdf.
+            point : dict
+                Parameter values to fix.                
+        """
+        mu, tau = draw_values([self.mu, self.tau], point=point)                                     
+        return np.sqrt(0.5 * tau / pi) * np.exp(-0.5 * tau * (x - mu) ** 2) 
```

Comment:
It part, the `pdf`/`cdf`/`random` methods are numpy/scipy based as ultimately (as I've started in another branch) I was heading towards a decorator which enables these methods to be called on an instance but also on the class as well (i.e., without class instantiation or model context) with specific numerical values:

```python
y = Normal.random(mu=0., sd=2)
```
So its basically just useful for *me* and I usually use it just for plotting or generating samples (which when done 'properly' using a model context is a bit slow on my computer), but the changes to make the `pdf` method theano based are probably trivial.